src/ui/profile_preferences.ui

```diff
@@ -373,7 +373,7 @@
           <item row="0" column="0">
            <widget class="QLabel" name="label_57">
             <property name="text">
-             <string>Chose spell check dictionary:</string>
+             <string>Choose spell check dictionary:</string>
```

Comment:
As only a single dictionary can be selected should it not be "Choose a spell-check dictionary:"?